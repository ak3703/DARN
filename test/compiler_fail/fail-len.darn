int main() {
	int[5][6] a;
	len(a);
}
