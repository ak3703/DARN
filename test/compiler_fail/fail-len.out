Fatal error: exception Failure("cannot get the length of non-1d-matrix")
